src/Raven.Studio/typescript/common/notifications/models/abstractNotification.ts

```diff
@@ -12,6 +12,8 @@ abstract class abstractNotification {
     title = ko.observable<string>();
     severity = ko.observable<Raven.Server.NotificationCenter.Notifications.NotificationSeverity>();
     type: Raven.Server.NotificationCenter.Notifications.NotificationType;
+    licenseLimitType = ko.observable<Raven.Server.Commercial.LimitType>();
```

Comment:
this field is calulated based on details - I think we shouln't introduce field used only in one case on *every* notification.

Reply:
Computed field?